src/components/Month/TransactionsSummary.tsx

```diff
@@ -8,7 +8,7 @@ import type {
 } from "../../types/transactionType/types";
 import { useParams } from "next/navigation";
 import { eventBus } from "@/utils/eventBus";
-import { TRANSACTION, MONTH } from "@/utils/eventsIds";
+import { TRANSACTION } from "@/utils/eventsIds";
```

Comment:
usually you can use just one letter 's' at the end in namings. For Example:
`NotesForStudentsIds` -> `NoteForStudentIds`